wiki/migrations/0005_remove_attachments_and_images.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.5 on 2017-09-07 19:03
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wiki', '0004_increase_slug_size'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='attachment',
+            name='current_revision',
+        ),
+        migrations.RemoveField(
+            model_name='attachment',
+            name='reusableplugin_ptr',
+        ),
+        migrations.RemoveField(
+            model_name='attachmentrevision',
+            name='attachment',
+        ),
+        migrations.RemoveField(
+            model_name='attachmentrevision',
+            name='previous_revision',
+        ),
+        migrations.RemoveField(
+            model_name='attachmentrevision',
+            name='user',
+        ),
+        migrations.RemoveField(
+            model_name='image',
+            name='revisionplugin_ptr',
+        ),
+        migrations.RemoveField(
+            model_name='imagerevision',
+            name='revisionpluginrevision_ptr',
+        ),
+        migrations.DeleteModel(
```

Comment:
Just curious - do Django's automatically-generated migrations remove each field like this before deleting models?

Reply:
I think these were just the foreign keys, removed separately to avoid referential integrity problems when dropping the tables.  As I noted in my other reply, this is a fairly tangled pair of models.